content/blog/stackstorm-simple-elegant-automation-for-everyone.md

```diff
@@ -0,0 +1,33 @@
+
+![](https://www.techworldwookie.com/images/stackstorm.png)
+
+Before we get too far, here is a quick re-cap of what StackStorm is. StackStorm uses Sensors, Rules, Triggers, Actions, and Workflows. You can think of it as If-this-then-that automation. Based on a event on some device, a StackStorm sensor can detect it, and this could trigger a Rule. The Rule would have some logic in it and if the trigger launched the Rule, the Rule would run an action, or a series of actions called a Workflow. It might sound a little involved but it is all packaged together to make it easy to deploy these solutions.
+
+![](https://www.techworldwookie.com/images/process.png)
+
+What if I told you there was a place called the StackStorm Exchange that already had 170 of these modules just waiting for you to plug into your projects? [Look at this link.](applewebdata://EBBB076B-5028-426E-9291-45124FE61477/exchange.stackstorm.com) It is quite an impressive stockpile of automation just waiting for you. I recently finished developing a StackStorm integration pack for HPE Primera Storage. Before that, I developed a pack to integrate HPE Nimble Storage as well. What communication can we develop? Let's take a look.
+
+I just needed an example of these two systems communicating. I settled on volumes. I wanted to be able to create a volume on the HPE Primera and have it **automatically** appear on a HPE Nimble storage array. Maybe this is something cool, maybe not, but it was just going to demonstrate the power of HPE devices talking to one another and have the capability to talk to the StackStorm integration packs out on the exchange. The process would look something like this:
+
+![](https://www.techworldwookie.com/images/flow.png)
+
+The most important thing to remember is that when I run a StackStorm workflow, one step in the process can be running a StackStorm action from one integration pack, and the next task can be a StackStorm action from any other pack! Now, I have StackStorm integration packs for HPE OneView, HPE Nimble, and HPE Primera. Not only can these three data center systems communicate with one another, they now have access to the StackStorm Exchange! 
+
+I have written StackStorm integration packs for **Qumulo**, **Aruba CX**, **Aruba Fabric Composer**, **iLo-amplifier**, and even one for **Arista Cloud Vision Portal**. The list has grown quite a bit since I started on this endeavour. If you have something you would like to have a StackStorm integration pack developed for, you can take action. You can start by reading my other blog posts [here:](https://developer.hpe.com/search/?term=stackstorm) You could check out the [Workshop-on-Demand](https://hackshack.hpedev.io/workshop/21) or take my free [StackStorm introductory training ](https://github.com/xod442/stackstorm-tutorial)and start your journey on your way to simple, elegant automation.
```

Comment:
Change "since I started on this endeavour" to:
"since I started this endeavour" (take out "on")

I think you could develop this summary a little further, talking a little more about the benefits one gets from making all these things talk to each other, how well the automation works and the benefits of automating in the first place. It could be just this one paragraph expanded or maybe split into two.

Reply:
paragraph has been enhanced! Thanks for the input